src/stores/GroupStore.js

```diff
@@ -46,9 +57,22 @@ export default class GroupStore extends EventEmitter {
         return this._summary;
     }
 
+    getGroupRooms() {
+        return this._rooms;
```

Comment:
Feels like we could do ‘groupRoomFromApiObject‘ in the store, maybe?

Reply:
That sounds like a good idea. Shall we descope it?

Reply:
yeah, fine by me